Ramda/Extensions/dec.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  dec.swift
+//  Ramda
+//
+//  Created by Christian Leovido on 28/09/2019.
+//
+
+import Foundation
+
+extension R {
+
+    /**
+
+     Decrements its argument.
+
+     - parameter value: Number to add to
+
+     - returns: Number + 1
+
+     See also inc
+
+     */
+    public class func dec<A: BasicArithmeticType>(_ number: A) -> A {
+
+        guard let one = 1 as? A else {
+            fatalError()
+        }
+
+        return number - one
```

Comment:
I don't think we should fatal error here. I would just return the original value if the value cannot be cast. What do they do in Ramda.js?

Reply:
I will remove the fatal error. They also support strings like "42", "42.4"